src/main/java/de/hs_mannheim/informatik/ct/util/MailAddress.java

```diff
@@ -0,0 +1,46 @@
+package de.hs_mannheim.informatik.ct.util;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.val;
+import lombok.var;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import java.util.Arrays;
+import java.util.List;
+
+@AllArgsConstructor @Getter
+public class MailAddress {
```

Comment:
Am I getting this right, that this class and the whole parsing is only done to decompose the mail address for the mail obfuscation used in the frontend? Wouldn't it be easier to decompose this manually in the properties file and access the elements from the frontend directly? (like so: https://stackoverflow.com/questions/56102116/access-application-properties-value-in-thymeleaf-template)

Reply:
yes, you could also enter the email address already split into the properties. I thought that might be a bit more comfortable in the configuration if the splitting happens automatically.